src/lib/viewers/image/ImageViewer.js

```diff
@@ -192,6 +197,20 @@ class ImageViewer extends ImageBaseViewer {
             // Since we are taking offsetWidth, we only need to apply the zoom to the width
             // as clearing the height will preserve the aspect ratio
             newWidth = type === 'in' ? width * IMAGE_ZOOM_SCALE : width / IMAGE_ZOOM_SCALE;
+
+            // We want to calculate the zoomedWidth / zoomedHeight to determine if we should toggle annotation mode to None
+            const ratio = Math.min(viewport.width / width, viewport.height / height);
+            const zoomedWidth = newWidth * ratio;
+            const zoomedHeight =
+                type === 'in' ? height * IMAGE_ZOOM_SCALE * ratio : (height / IMAGE_ZOOM_SCALE) * ratio;
+
+            // If the image is overflowing the viewport, we should set annotation mode to be NONE so that the user can pan
+            if (zoomedWidth > viewport.width || zoomedHeight > viewport.height) {
+                if (this.options.enableAnnotationsImageDiscoverability) {
+                    this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.CANCEL));
+                    this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
```

Comment:
Is this necessary? Seems like `processAnnotationModeChange` should already handle it

Reply:
Nope this isn't needed, will remove.